openlcs/packages/serializers.py

```diff
@@ -69,16 +70,28 @@ class Meta:
                   "copyright_detections"]
 
     def get_license_detections(self, obj):
```

Comment:
With the output update, related docstring needs an update.

Reply:
Oh, when I say the docstring, I mean that of sources endpoints, it still displays as below:
```
        "source": {
            "id": 5,
            "checksum": "9fb25c78a1aa98a193d9d8b438113624456042f58b3061c851d1f9536d3046b7",
            "name": "rpm-libs-4.14.3-23.el8",
            ...
            "license_detections": [],
            "copyright_detections": []
        ....
```

Reply:
DONE